flow360/component/geometry.py

```diff
@@ -289,6 +289,26 @@ def show_available_groupings(self, verbose_mode: bool = False):
             show_ids_in_each_group=verbose_mode,
         )
 
+    @classmethod
+    # pylint: disable=redefined-builtin
+    def from_local_storage(cls, id: str = None, local_storage_path="") -> Geometry:
```

Comment:
Pylint here is complaining the `id` being redefined. Can we use something that does not conflict? Like geometry_id?

I know we have other places that did not enforce this... But let's not introduce new ones :)